website/static/js/registrationModal.js

```diff
@@ -52,14 +52,15 @@ var RegistrationViewModel = function(confirm, prompts, validator) {
 
     var validation = [{
         validator: function() {
-            var endEmbargoDateTimestamp = self.embargoEndDate().getTime();
+            var timeZoneOffset = self.embargoEndDate().getTimezoneOffset() * (60 * 1000);
+            var endEmbargoDateTimestamp = self.embargoEndDate().getTime() + timeZoneOffset;
             return endEmbargoDateTimestamp > TWO_DAYS_FROM_TODAY_TIMESTAMP;
         },
         message: 'Embargo end date must be at least three days in the future.'
     }, {
 	validator: function() {
             var endEmbargoDateTimestamp = self.embargoEndDate().getTime();
-	    return endEmbargoDateTimestamp < FOUR_YEARS_FROM_TODAY_TIMESTAMP;
+	        return endEmbargoDateTimestamp < FOUR_YEARS_FROM_TODAY_TIMESTAMP;
```

Comment:
It looked wrong on my end, so I changed something. Let me go back and fix that, because you're definitely right. That doesn't look right at all.